src/components/Login/Login.js

```diff
@@ -1,29 +1,72 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { withFirebase } from '../Firebase/context';
+import PropTypes from 'prop-types';
+import * as ROUTES from '../../constants/routes';
+
 import './Login.css';
-function Login() {
-  return (
-    <div className = "LoginAll">
-      <form method="post">
-        <div className = "inputCenter">
-          <div className = "inputLogin">
-            <label>
-                Email
-              <br></br>
-              <input type="email" name="email" placeholder="Email address"/>
-            </label>
-            <br></br><br></br>
-            <label>
-                Password
+
+const INITIAL_STATE = {
+  email: '',
+  password: ''
+};
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { ...INITIAL_STATE };
+  }
+
+  onSubmit = (event) => {
+    event.preventDefault();
+    const { email, password } = this.state;
+
+    this.props.firebase.login(email, password).then(() => {
+ 
+      this.props.history.push(ROUTES.MAIN);
```

Comment:
We don't actually need to change route here, as it can be done in session when new user has been signed in